webview-ui/src/components/settings/providers/OpenRouter.tsx

```diff
@@ -81,7 +81,18 @@ export const OpenRouter = ({
 			<div className="text-sm text-vscode-descriptionForeground -mt-2">
 				{t("settings:providers.apiKeyStorageNotice")}
 			</div>
-			{!apiConfiguration?.openRouterApiKey && (
+			<VSCodeTextField
+				value={apiConfiguration?.openRouterApiKeyEnvVar || ""}
+				type="text"
+				onInput={handleInputChange("openRouterApiKeyEnvVar")}
+				placeholder={t("settings:placeholders.openRouterApiKeyEnvVar")}
```

Comment:
I don't think this is the right translation key

Reply:
thanks for catching -- fixed here and other providers